src/components/sky-nav/sky-nav.test.ts

```diff
@@ -21,7 +21,30 @@ const initSkyNavJS = (utils: PleasantestUtils, navButton: ElementHandle) =>
 test(
   'can be opened on small screens',
   withBrowser({ device: iPhone }, async ({ utils, screen, user, page }) => {
-    await utils.injectHTML(await skyNavMarkup({ includeMainDemo: true, menu }));
+    // The rendered markup needs to be captured so we can pass it into the
+    // `page.evaluate` function below
+    const renderedMarkup = await skyNavMarkup({
+      includeMainDemo: true,
+      menu,
+    });
+    // Pleasantest uses `document.innerHTML` to inject the markup into the DOM,
+    // but that means inline scripts are not executed.
```

Comment:
Probably worth linking the issue?
```suggestion
    // but that means inline scripts are not executed.
    // https://github.com/cloudfour/pleasantest/issues/526
```